test/integration/reputation/iprepd.yaml

```diff
@@ -0,0 +1,15 @@
+---
+listen: 0.0.0.0:8080
```

Comment:
@ameihm0912 is it possible to please pick a less common port?

Reply:
No problem, I'll adjust this PR